sql/moz-fx-data-shared-prod/telemetry_derived/unified_metrics_v1/query.sql

```diff
@@ -0,0 +1,190 @@
+WITH unioned AS (
+  SELECT
+    *,
+    'Fenix' AS normalized_app_name
+  FROM
+    fenix.clients_last_seen_joined
```

Comment:
We've just added `clients_last_seen_joined` tables which joins selected `metrics` and `baseline_clients_last_seen`